client/src/modules/vouchers/toolkit/patient_fled/patient_fled.js

```diff
@@ -0,0 +1,193 @@
+    cashboxRow.debit = vm.totalSelected;
+    cashboxRow.credit = 0;
+    rows.push(cashboxRow);
+
+
+    // then loop through each selected item and credit it with the convention account
+    invoices.forEach(invoice => {
+      const row = ToolKits.getBlankVoucherRow();
+
+      row.account_id = conventionAccountId;
+      row.reference_uuid = invoice.uuid;
+      row.entity_uuid = invoice.entity_uuid;
+      row.credit = invoice.balance;
+
+      // this is needed for a nice display in the grid
+      row.entity = {
+        label : invoice.entityReference,
+        type : 'D',
+        uuid : invoice.entity_uuid,
+      };
+
+      // this is need to display references in the grid nicely
+      row.document = {
+        uuid : invoice.uuid,
+        reference : invoice.reference,
+        document_type : 'VOUCHERS.GLOBAL.PATIENT_FLED',
+      };
+
+      // add the row in to the
+      rows.push(row);
+    });
+
+    return rows;
+  }
+
+  /* ================ Invoice grid parameters ===================== */
+  vm.gridOptions = {
+    enableFiltering : true,
+    enableColumnMenus : false,
+    fastWatch : true,
+    flatEntityAccess : true,
+    enableSelectionBatchEvent : false,
+    onRegisterApi,
+  };
+
+  vm.gridOptions.columnDefs = [{
+    field : 'reference',
+    displayName : 'TABLE.COLUMNS.REFERENCE',
+    headerCellFilter : 'translate',
+  }, {
+    field : 'date',
+    type : 'date',
+    cellFilter : 'date',
+    displayName : 'TABLE.COLUMNS.BILLING_DATE',
+    headerCellFilter : 'translate',
+    enableFiltering : false,
+  }, {
+    field : 'balance',
+    type : 'number',
+    displayName : 'TABLE.COLUMNS.COST',
+    headerCellFilter : 'translate',
+    enableFiltering : true,
+    cellFilter : 'currency:'.concat(Session.enterprise.currency_id),
+    cellClass : 'text-right',
+  }];
+
+  function onRegisterApi(gridApi) {
+    vm.gridApi = gridApi;
+    vm.gridApi.selection.on.rowSelectionChanged(null, rowSelectionCallback);
+  }
+
+  // helper function for the rowSelectionCallback()
+  function aggregate(sum, row) {
+    return sum + row.balance;
+  }
+
+  // called whenever the selection changes in the ui-grid
+  function rowSelectionCallback() {
+    const selected = vm.gridApi.selection.getSelectedRows();
+    const aggregation = selected.reduce(aggregate, 0);
+
+    vm.hasSelectedRows = selected.length > 0;
+    vm.totalSelected = Number.parseFloat(aggregation.toFixed(MAX_DECIMAL_PRECISION));
+  }
+
+  /* ================ End Invoice grid parameters ===================== */
+
+  // submission
+  function submit(form) {
+    if (form.$invalid) { return; }
+
+    const selected = vm.gridApi.selection.getSelectedRows();
+
+    const bundle = generateTransactionRows({
+      cashbox    : vm.cashbox,
+      convention : vm.convention,
+      invoices   : selected,
+    });
+
+    Instance.close({
+      rows       : bundle,
+      convention : vm.convention,
+      type_id    : bhConstants.transactionType.CONVENTION_PAYMENT,
```

Comment:
This isn't a CONVENTION_PAYMENT since it is an expense.  A better type would be [GENERIC_EXPENSE](https://github.com/IMA-WorldHealth/bhima/blob/master/client/src/js/constants/bhConstants.js#L70).

This is mentioned in the issue:
> The voucher type would be set to autre charge perte or something similar.